pkgs/top-level/python-packages.nix

```
@@ -6237,6 +6237,16 @@ self: super: with self; {
 
   migen = callPackage ../development/python-modules/migen { };
 
+  litex = callPackage ../development/python-modules/litex { };
+
+  litex-boards = callPackage ../development/python-modules/litex-boards { };
+
+  litespi = callPackage ../development/python-modules/litespi { };
+
+  pythondata-cpu-vexriscv = callPackage ../development/python-modules/pythondata-cpu-vexriscv { };
+  pythondata-software-picolibc = callPackage ../development/python-modules/pythondata-software-picolibc { };
+  pythondata-software-compiler_rt = callPackage ../development/python-modules/pythondata-software-compiler_rt { };
```

```suggestion
  pythondata-cpu-vexriscv = callPackage ../development/python-modules/pythondata-cpu-vexriscv { };

  pythondata-software-picolibc = callPackage ../development/python-modules/pythondata-software-picolibc { };

  pythondata-software-compiler-rt = callPackage ../development/python-modules/pythondata-software-compiler-rt { };
```

Per python ecosystem docs we always normalise attrs and pnames and replace special characters like underscore with a dash and make it all lower cased.